lib/tasks/colormaps.rake

```diff
@@ -35,6 +35,19 @@ namespace :colormaps do
 
     exit 1 if error_count > 0
 
+    # explicitly invoke the task here to make sure task colormaps:load completes successfully.
+    Rake::Task['colormaps:log_colormaps_load'].invoke
+
+  end
+
+  desc "Record the last run of task 'colormaps:load' by touching a file in ./tmp dir"
+  task :log_colormaps_load do
+    Dir.mkdir Rails.root.join('tmp') unless Dir.exist? Rails.root.join('tmp')
+    Dir.glob(Rails.root.join('tmp', "colormaps_load_*")).each do |f|
+      File.delete(f)
+    end
+
+    FileUtils.touch Rails.root.join('tmp', "colormaps_load_#{Time.now.to_i}")
```

Comment:
Note that if you're using touch, you don't need to append the date, you can just use `File.mtime(Rails.root.join('tmp', 'colormaps_load')` to get the modification time. This could simplify logic.